src/Azs.Subscriptions.Admin/readme.md

```diff
@@ -17,7 +17,7 @@ This directory contains the PowerShell module for the SubscriptionsAdmin service
 This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
 
 ## Module Requirements
-- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 1.6.0 or greater
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 1.7.4 or greater
```

Comment:
This statemetn is wrong, but no need to fix at the moment

Reply:
any idea why autorest is modifying this statement?